README.md

```diff
@@ -79,23 +79,14 @@ The EF-CMS is comprised of two components: the API and the UI. Both must be run
 
 - Node v10.15.3
 - npm v6.4.1
-- Ghostscript v9.23 (see Setup below)
 - ClamAV v0.101.2 (see Setup below)
 
 ### Setup
 
-Follow [the installation prerequisites for Ghostscript4JS](https://www.npmjs.com/package/ghostscript4js#prerequisites). macOS users can do the following:
```

Comment:
@waldoj I'd like to chat a little more about this. That success message should return when `localhost:1234` is reachable (the script iteratively makes a curl request, looking for a 200 response, and then returns). Is it _actually_ returning this success message before the site is reachable? Or is it simply returning a success / things are done message out of context of the setup process as a whole?

Reply:
I'm thinking of this in terms of UX, and have two concerns along those lines. The first is that the success message scrolls past so fast that you'd have to be paying _real_ close attention to spot it. I rarely see it. The second is that a reasonable person would run the build command and keep checking back until it's done, at which point, they'd think "OK...now what?" We want to give them that next step at that point. Right now, they'd need to scroll up 1,805 lines to find the next step.

I'd like `./docker-run.sh` to go back to concluding with an instruction about what the user should do next, although I'm certainly open to alternate methods of achieving the same outcome.

Reply:
Based on a short conversation with Matt, this looks to be more than a simple fix. Because this is not urgent, this will be resolved in a future sprint.